### ISSUE TITLE
Flux is always reset in adv_tra_vert_ppm

### ISSUE THREAD

**Issue by goord** [FESOM/fesom2#82]
Irrespective of the ``init_zero`` option, the flux array is always set to start with zeros in this routine. Intentionally?

**Comment by dsidoren** [comment]
hi, not. is a bug! luckily PPM shall not be used in combination with FCT but now I know why it did not work. THANK YOU!

**Comment by goord** [comment]
Hi @dsidoren the test case I have does use PPM in combination with FCT. If this is not a typical case, which vertical tracer advection scheme should we use in combination with FCT?

**Comment by koldunovn** [comment]
@dsidoren Maybe you can make a PR with the fix before we forget? :)

**Comment by koldunovn** [comment]
@goord Currently we use:

```
tra_adv_hor ='MFCT'  
tra_adv_ver ='QR4C' 
tra_adv_lim ='FCT'  
```
As a default, this covers most of our use cases.

**Comment by dsidoren** [comment]
but you can use PPT as well (although it limits by itself).

**Comment by JanStreffing** [comment]
closed by #95
